web/pdf_viewer.js

```diff
@@ -312,6 +316,7 @@ class PDFViewer {
       this.removePageBorders = options.removePageBorders || false;
     }
     this.maxCanvasPixels = options.maxCanvasPixels;
+    this.enableDetailCanvas = options.enableDetailCanvas;
```

Comment:
The default value should agree with `AppOptions`.
`this.enableDetailCanvas = options.enableDetailCanvas !== false;`